### DISCUSSION TITLE
Going from P7 to P8?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1380] [Q&A] [answered]
Is there a document detailing the changes required in going from P7 to P8, beyond building the new executable? Things like changes to input.nml, input files, etc? I looked [here](https://docs.google.com/spreadsheets/d/1ZMrfBF03bBj6CIW9EQJ6MNfPuQjecilxa7qrV8k2660/edit?usp=sharing) but it seems like it has not been updated for P8.

If someone could also point me to a full run directory of P8 on Orion then I could likely work out the necessary changes myself, but documentation is always appreciated. :)

**Comment by JessicaMeixner-NOAA** [comment, marked as answer]
@benjamin-cash The spreadsheet has not been updated for P8.  You can run the workflow on orion using the tag (see the Wednesday coupled meeting notes for information on tags, instructions on running, etc) to get the run directory you are looking for.  The spreadsheet will be updated at some point soon, but no guaranteed timeline on that.

**Reply by benjamin-cash** [reply]
Hi @JessicaMeixner-NOAA - Thanks for the suggestion! I was able to create a run directory and tar up everything, including the linked files, for move to Frontera. It would be good to see the updated spreadsheet at some point or some other detailing of the P8 configuration, just because I am a complete novice with global-workflow and am not really certain I correctly set up a P8 experiment.

**Comment by benjamin-cash** [comment]
Another P8 question - Has the formula for calculating how many pes are allotted to each component changed in P8? Previously I have allocated `6*layout_x*layout_y` to the mediator. But looking at nems.configure for the P8 run I generated on Orion that is now the number allocated to CHM_model (6x8x8 for 384) , and MED_model is only allocated 300. How is that 300 determined?

[Cross posted with UFS-R2O-Coupled slack]
